keypair/Cargo.toml

```diff
@@ -18,7 +18,7 @@ solana-derivation-path = { workspace = true, optional = true }
 solana-pubkey = { workspace = true }
 solana-seed-derivable = { workspace = true, optional = true }
 solana-seed-phrase = { workspace = true }
-solana-signature = { workspace = true, features = ["verify"] }
+solana-signature = { workspace = true, features = ["std", "verify"] }
```

Comment:
The "std" feature is enabled by default, just checking if that was expected or not since it is being explicitly enabled here.